exercises/practice/hello-world/.meta/config.json

```diff
@@ -1,13 +1,8 @@
 {
-  "blurb": "The classical introductory exercise. Just say \"Hello, World!\"",
+  "blurb": "The classical introductory exercise. Just say \"Hello, World!\".",
   "authors": [
     "benreyn"
   ],
-  "contributors": [
-    "agentofuser",
-    "NobbZ",
-    "yurrriq"
-  ],
```

Comment:
What was the reason for removing these contributors?

Reply:
Hmm. I must have messed up something git-wise. On my branch, hello-world was missing a few files. As you can see in this commit [`4a5d6a0` (#139)](https://github.com/exercism/lfe/pull/139/commits/4a5d6a07a81816da93bcca8f4379d95022e55196) it adds them, not edits them.

I was convinced those files are missing on main too. Let me fix this.

Reply:
Sure